pkg/terraform/exec/plugins/vsphereprivate/resource_vsphereprivate_import_ova.go

```diff
@@ -189,11 +198,47 @@ func findImportOvaParams(client *vim25.Client, datacenter, cluster, datastore, n
 	for _, hostObj := range hosts {
 		foundDatastore := false
 		foundNetwork := false
-		err := hostObj.Properties(ctx, hostObj.Reference(), []string{"network", "datastore", "runtime"}, &hostSystemManagedObject)
+		err := hostObj.Properties(ctx, hostObj.Reference(), []string{"config.product", "network", "datastore", "runtime"}, &hostSystemManagedObject)
 		if err != nil {
 			return nil, err
 		}
 
+		// If HardwareVersion is 13 there is no reason to continue checking
+		// There is a ESXi host that does not support hardware 15.
```

Comment:
I'm not sure to understand that comment. Are we talking about all the older [ESXi hosts 6.7 and earlier not supporting Hardware version 15](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-789C3913-1053-4850-A0F0-E29C3D32B6DA.html)?